docs/ref/models/querysets.txt

```diff
@@ -2431,10 +2431,10 @@ This has a number of caveats though:
 * The model's ``save()`` method will not be called, and the ``pre_save`` and
   ``post_save`` signals will not be sent.
 * It does not work with child models in a multi-table inheritance scenario.
-* If the model's primary key is an :class:`~django.db.models.AutoField`, the
-  primary key attribute can only be retrieved on certain databases (currently
-  PostgreSQL, MariaDB, and SQLite 3.35+). On other databases, it will not be
-  set.
+* If the model's primary key is an :class:`~django.db.models.AutoField` and
+  ``ignore_on_conflict`` is False, the primary key attribute can only be
```

Comment:
```suggestion
  ``ignore_conflicts`` is False, the primary key attribute can only be
```

Reply:
@simkimsia If you don't have time, I can update it.

Reply:
Yes please thank you 🙇🏻‍♂️

I'm away from keyboard